src/plugins/links.js

```diff
@@ -69,9 +69,17 @@ export function linkBubble(options) {
 		}),
 
 		appendTransaction: (transactions, oldState, state) => {
+			// Don't open bubble at editor initialisation
+			if (oldState?.doc.content.size === 2) {
```

Comment:
This will also be in effect if you paste into an empty document. I can't think of any case where that's a problem. Just wanted to mention.

Reply:
Uuhh... what if you insert a link via the link picker on an empty document. Actually would be nice if that behaved exactly as in a doc that already contains some characters. Not so important - but still a bit inconsistent.

Oh... doesn't the initial load happen through the `ydoc`? That would be caught by the other conditions later on - wouldn't it?